src/modules/savemanager/savemanager-service.js

```diff
@@ -340,6 +340,15 @@ angular.module('anol.savemanager')
                 this.activePollingIntervals[layerName] = $interval(function() {
                     var url = self.pollingUrl + '?layer=' + layerName;
                     $http.get(url).then(function(resp) {
+                        // Backend returns error pages for failed requests
+                        // so we have to check for the content-type additionally.
+                        if (resp.headers('Content-Type') !== 'application/json') {
```

Comment:
Doesn't it return an error status code? That might be nicer to check for.

Reply:
Unfortunately not. Currently, there is a global error handler configured, that reacts to status codes and returns html with 200.